### PR TITLE
Cicerone wik 14 leaflet routing machine

### PR DESCRIPTION
**Pull request by facuzero**
Routing machine works fine but i need learn tailwind to fix the visual appearance

### CODE REVIEW

File: pages/places/_id/place-details.vue [hunk 1]
```diff
@@ -61,10 +76,27 @@ export default {
     }
   },
   mounted () {
-    this.fetchWiki()
```
Comment (njrzr) [inline pages/places/_id/place-details.vue:64]: @facuzero why you deleted this.fetchWiki(), this method is used to get the images and excerpt to show on the view, the this.isLoading can be deleted.

**Review by njrzr — CHANGES_REQUESTED** [review]
@facuzero read the comments I did, and fix the issues I found.

File: pages/places/_id/place-details.vue (hunk 1 above)
Comment (facuzero) [reply pages/places/_id/place-details.vue:64]: @njrzr sorry, i fixed up

**Review by njrzr — CHANGES_REQUESTED** [review]
@facuzero it's all good, I will request one more change, try to make the map height taller, the height seems small.

**Comment by facuzero** [comment]
@njrzr solved the problem about the size of map

**Review by njrzr — APPROVED** [review]

**Pull request merged**